mne/gui/_file_traits.py

```diff
@@ -285,14 +287,40 @@ def _get_n_omitted(self):
     @cached_property
     def _get_inst(self):
         if self.file:
-            info = read_info(self.file, verbose=False)
-            if info['dig'] is None:
+            try:
+                info = read_info(self.file, verbose=False)
+            except ValueError:  # try reading dig_montage
```

Comment:
How about just logging the error message before looking for DigMontage? I don't like the idea of changing the behavior of the public function.